website/docs/cli.md

```diff
@@ -84,6 +84,15 @@ Compiles your site for production.
 | `--bundle-analyzer` | `false` | Analyze your bundle with the [webpack bundle analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer). |
 | `--out-dir` | `build` | The full path for the new output directory, relative to the current workspace. |
 | `--no-minify` | `false` | Build website without minimizing JS/CSS bundles. |
+| `--use-old-css-minifier` | `false` | Use only default preset cssnano for CSS minification (see info below for more details). |
```

Comment:
if we want to keep this over time, and maybe later change or add new minifier logic, what about giving a proper name to each config? like "simple" vs "advanced" or something?

boolean + the "old" term do not look very extensible to me